ocpmodels/models/utils/embeddings/elemental_embeddings.py

```diff
@@ -3,6 +3,9 @@
 
 This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
+
+
+Original CGCNN k-hot elemental embeddings.
```

Comment:
Should we rename this file to `khot_embeddings` and `forcenet_embeddings` to `continuous_embeddings`?